src/seedsigner/models/settings_definition.py

```diff
@@ -711,6 +842,15 @@ class SettingsDefinition:
                       selection_options=SettingsConstants.ALL_DISPLAY_CONFIGURATIONS,
                       default_value=SettingsConstants.DISPLAY_CONFIGURATION__ST7789__240x240),
 
+        SettingsEntry(category=SettingsConstants.CATEGORY__SYSTEM,
+                      attr_name=SettingsConstants.SETTING__HARDWARE_CONFIG,
+                      abbreviated_name="hw_conf",
+                      display_name=_mft("Hardware Configuration"),
+                      type=SettingsConstants.TYPE__SELECT_1,
+                      visibility=SettingsConstants.VISIBILITY__HARDWARE,
+                      selection_options=SettingsConstants.ALL_HARDWARE_PIN_CONFIGS,
+                      default_value=SettingsConstants.HARDWARE__RPI_40),
+
```

Comment:
What are the odds that a user will be able to configure this setting if it's incorrect?

Or worse: what if a user with a working hardware setup AND persistent settings enabled changes it?!

I think so far it's an unsolved problem how we'll configure a release image / SD card for a particular target hardware.

Reply:
I agree. I think we actually need a different kind of settings config file. 

I noticed you made some changes previously to allow hot swapping displays, but I'm not really convinced we would ever need that. You would turn off the power when wiring in a different display. Therefore the mode it starts up in (gpio mapping, display hardware, embedded Linux platform) should be a constant.

It was implemented like this for convenience while prototyping.